File: python/nivel 4 trampas/23.py
"""
Ejercicio 23
Convertir un número decimal a binario
Sin usar funciones
"""
numero = int(input("Ingresa un numero decimal: "))

if numero == 0:
  print("Binario: 0")
else:
  binario = ""

  while numero > 0:
    resto = numero % 2
    binario= str(resto) + binario
    numero //=2

print("Binario: ", binario)

File: javascript/nivel 4 trampas/23.js
/*
Ejercicio 23
Convertir un número decimal a binario (sin usar funciones)
*/
let numero = parseInt(prompt("Ingrese un numero decimal: "));

let binario = "";

if(numero === 0){
  binario= 0;
}else{
  while(numero > 0){
    let resto = numero % 2;
    binario = resto + binario;
    numero = Math.floor(numero/2);
  }
}

console.log("El numero en binario es: ", binario);